### DISCUSSION TITLE
Next/Link not loading fixed path routes?

### DISCUSSION THREAD

**Discussion by mathewtrivett** [vercel/next.js#16388] [Help]
Hi I was wondering if someone could give a bit of help?

We're using Next and Next/Link for a headless Wordpress site.

Some pages are dynamic, routed from the pages directory. Others used a fixed route type page. See an example directory structure below:

```
pages/
    news/
       overview.jsx
       [slug].jsx
```

Imagine we a menu with links that look like this:

```
<Link href=`/news/overview` />
<Link href={`/news/${item.slug}`} />
```

If I'm on the `/news/a-slug` page, when I click on `/news/overview` it pushes that URL into browser history but doesn't actually navigate me there. Am I doing something wrong here?

Can you use Next/Link for pages that don't have a dynamic component?

**Comment by jamesmosier** [comment]
The example you outlined *should* work. I tested it and when I am on the `news/[slug]` page and click a `<Link>` to go to `news/overview` it navigates fully and shows the "overview" content.

Do you happen to have a repository you could share?

---

Also the link for ```<Link href={`/news/${item.slug}`}>``` should be ```<Link href="/news/[slug]" as={`/news/${item.slug}`}>``` since it is a [dynamic route](https://nextjs.org/docs/api-reference/next/link#dynamic-routes).

Although if you install `next@canary` then ```<Link href={`/news/${item.slug}`}>``` works as it is written 😄
